FirstProject/new-evaluation.html

```diff
@@ -0,0 +1,536 @@
+                    <h4 style="color: white;">Junior</h4>
+                </div>
+                <div class="col-md-3">
+                    <h4 style="color: white;">Middle</h4>
+                </div>
+                <div class="col-md-3">
+                    <h4 style="color:white;">Senior</h4>
+                </div>
+        </div>
+
+        <div class="row_section" style="border: 1px solid black;">
+            <div class="col-md-3">
+                <ul>
+                    <li class="bullets"><p>&#9711;</p></li>
+                </ul>
+            </div>
+            <div class="col-md-3">
+                <ul style="display: inline-flex;">
+                    <li class="bullets"><p>&#9711;</p></li>
+                    <li class="bullets"><p>&#9711;</p></li>
+                    <li class="bullets"><p>&#9711;</p></li>
+                </ul>
+            </div>
+            <div class="col-md-3">
+                <ul style="display: inline-flex;">
+                    <li class="bullets"><p>&#9711;</p></li>
+                    <li class="bullets"><p>&#9711;</p></li>
+                    <li class="bullets"><p>&#9711;</p></li>
+                </ul>
+            </div>
+            <div class="col-md-3">
+                <ul style="display: inline-flex;">
+                    <li class="bullets"><p>&#9711;</p></li>
+                    <li class="bullets"><p>&#9711;</p></li>
+                    <li class="bullets"><p>&#9711;</p></li>
+                </ul>
+            </div>
+        </div>
+
+        <div class="row">
+            <h2 class="description_section" style="color: cornflowerblue;">Should the candidate be hired?</h2>
+            <textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px;" placeholder="The type of project that is suitable for the candidate
+Is guidance required for the candidate
+" ></textarea>
+        </div>
+
+        <div class="row">
+            <h2 class="description_section" style="color: cornflowerblue;">General Impression</h2>
+            <textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px;" placeholder="*required
+" ></textarea>
+        </div>
+
+        <div class="row">
+            <h2 class="description_section" style="color: cornflowerblue;">Workflow, Leadership & Soft Skills</h2>
+            <textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px;" placeholder="Describe the environment in which the candidate works.
+Describe any guidance or management experience
+" ></textarea>
+        </div>
+
+        <form class="language_form" style="text-align:center;">
+            <fieldset class="field_section" style="display:inline;">
+                <legend>OOP, Design Patterns:</legend>
+                    <ul style="display: inline-flex; text-align: right">
+                        <li>
+                            Classes <select>
+                            <option selected disabled>Evaluation</option>
```

Comment:
If only there was a way to do this so we didn't repeat so much markup all over the place... :P